extensions/localPublish/src/index.ts

```diff
@@ -130,6 +130,19 @@ class LocalPublisher implements PublishPlugin<PublishConfig> {
 
   private publishAsync = async (botId: string, version: string, fullSettings: any, project: any, user) => {
     try {
+      let port;
```

Comment:
Needed to move the port selection up in the code a bit so that it can be passed to the build step.